### PR TITLE
update readme with installation steps

### PR DESCRIPTION
**Pull request by SimonLab**
ref: #78
add dwylbot installation steps to the Readme

### CODE REVIEW

**Review by samhstn — APPROVED** [review]
Looks great @SimonLab, really coming together

File: README.md [hunk 1]
```diff
@@ -64,6 +64,24 @@ We refer to this as our "Workflow".
...
+- click on "Manage dwylbot installations" button. This will open the _dwylbot_ github app page.
+  ![configure-dwylbot](https://user-images.githubusercontent.com/6057298/27295764-fadffbb2-5515-11e7-85fb-dc267e462c15.png)
+
+- click ```configure``` to select the organisations where you want dwylbot active on
+  ![configure-orgs](https://user-images.githubusercontent.com/6057298/27295790-0e296884-5516-11e7-9840-6e41c153e67c.png)
+
+- you can install _dwylbot_ on all the repositories of the organisation or you can choose some specific repositories
+  ![select-repos](https://user-images.githubusercontent.com/6057298/27295823-2971c898-5516-11e7-88e0-6e37e7acd5ba.png)
+
+- click ```install```. _dwylbot_ is now installed :tada: Just use as normal Github, _dwylbot_ will help you and your team to enhance your workflow.
```
Comment (nelsonic) [inline README.md]: consider revising grammar in this sentence.

**Review by nelsonic — APPROVED** [review]
@SimonLab great instructions!
@naazy are you trying them out this morning?
Would be _really_ good to get your feedback on the process. thanks! 🎉

**Comment by naazy** [comment]
Have just installed and the instructions were super easy and clear :smile:

🤖

**Comment by nelsonic** [comment]
@naazy thanks for confirming! 🎉

**Pull request merged**